app/latigo/time_series_api/prediction_storage_provider.py

```diff
@@ -97,7 +97,7 @@ def put_prediction(self, prediction_data: PredictionDataSet) -> Dict[Tuple[str,
         skipped_values = 0
         stored_values = 0
         skipped_tags = 0
-        logger.info(f"Storing predictions for %s columns (before filtering)", len(df.columns))
+        logger.info(f"Storing predictions for {len(df.columns)} columns (before filtering)")
```

Comment:
previous version was right. Logger prefer passing to it params instead of string formatting.
```suggestion
        logger.info("Storing predictions for %s columns (before filtering)", len(df.columns))
```

Reply:
previous variant woult print next:
"('Storing predictions for %s columns (before filtering)', 6)" - Is this right?